packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/add/add.controller.js

```diff
@@ -20,6 +17,11 @@ export default class {
       monthlyBilling: false,
       autoscaling: this.autoscaling,
     };
+    this.constants = {
+      nameMinLength: 2,
```

Comment:
Can you maintain all these lengths and reg ex in constant file which is in same folder ?

Reply:
Done.